SRT to VTT Converter/Utils.cpp

```diff
@@ -45,14 +46,54 @@ void Utils::openFile(const string& filepath, wifstream& stream)
 			throw runtime_error("Converting UTF-16 encoded files is not supported on your platform.");
 		#endif
 
-		// ASCII, ANSI, UTF-8, none (treat as UTF-8)
-		default:
+		case AutoIt::TextEncodingDetect::UTF8_BOM:
+		case AutoIt::TextEncodingDetect::UTF8_NOBOM:
 			stream.open(filepath);
 			stream.imbue(locale(fin.getloc(), new codecvt_utf8<wchar_t, 0x10ffff, consume_header>));
 			break;
+
+		// ASCII, ANSI, none
+		default:
+			stream.open(filepath);
+			stream.imbue(locale(fin.getloc(), new codecvt<wchar_t, char, mbstate_t>));
+			break;
 	}
 }
 
+void Utils::dumpEncodingType(const unsigned char *pBuffer, size_t size)
+{
+	AutoIt::TextEncodingDetect encodingDetector;
+	switch (encodingDetector.DetectEncoding(pBuffer, size)) {
+		case AutoIt::TextEncodingDetect::None:
+			printf(" No text encoding detected.  ");
```

Comment:
Please include `iostream` right under `dirent.h` ([line 17](https://github.com/woollybogger/srt-to-vtt-cl/pull/3/files#diff-9e117f3fabea37bc691445b939ba8928R17)) and use `cout` instead of `printf`.